raiden/api/python.py

```diff
@@ -79,6 +79,23 @@ def event_filter_for_payments(
     return sent_and_target_matches or received_and_initiator_matches
 
 
+def transfer_data_from_transfer_task(transfer_task):
+    role = views.role_from_transfer_task(transfer_task)
+
+    if role == 'initiator':
+        transfer_data = transfer_task.manager_state.initiator.transfer_description.to_dict()
+    elif role == 'mediator':
+        transfer_data = transfer_task.mediator_state.to_dict()
+        transfer_data['number_of_pairs'] = len(transfer_data.pop('transfers_pair'))
+    elif role == 'target':
+        transfer_data = transfer_task.target_state.transfer.to_dict()
+    else:
+        return None
+
+    transfer_data['role'] = role
+    return transfer_data
```

Comment:
I think we should standardize the output of the API... a transfer is actually different than a transfer description and may contain more or less attributes. We might not want to show everything here.